.github/workflows/build.yml

```diff
@@ -11,6 +11,7 @@ jobs:
     name: Build
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
```

Comment:
**suggestion (history):** you might want to pull this out into its own commit.

Allowing all builds in a matrix to run [even if one of them failed](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-preventing-a-specific-failing-matrix-job-from-failing-a-workflow-run) is useful in of itself (and has way lower impact), so I think it would make sense to decouple it from the .NET 5 support work. That way, we could bring it in sooner, if need be.

Reply:
Makes sense, but no other 1.0 PRs pending atm so if people happy with this PR it can go in in this PR sooner rather than later. Otherwise, I can do a separate PR for this.